backend/src/main/resources/web/openapi.yml

```diff
@@ -46,27 +46,9 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Tiltakstype"
-    post:
-      tags:
-        - mulighetsrommet
-      operationId: createTiltakstype
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: "#/components/schemas/UnsavedTiltakstype"
-      responses:
-        201:
-          description: The created tiltakstype
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Tiltakstype"
-
-  /api/tiltakstyper/{id}:
+  /api/tiltakstyper/{tiltakskode}:
```

Comment:
Her kan frontend også oppdateres.

`npm run generate-api-client` for å generere opp ny klient, og tilsvarende enderinger basert på compile errors i applikasjonen etter dette. 

(Synes forøvrig det kan gi mening å trekke ut denne api-klienten til en separat pakke for å skille på generert bibliotekskode og applikasjonskode i frontend, men det blir en annen oppgave.)